service/crawlingKeywordService.js

```diff
@@ -26,9 +26,8 @@ const getCrawlingKeyword = async (req, res) => {
     if (hasKeyword) {
       return res.status(200).json({
         url: req.params.url,
-        urlLink: req.params.url,
         hasKeyword: hasKeyword,
-        urlText: innerText,
+        urlText: innerText.replace(/\n|\r|\t/g, ""),
       });
     } else if (innerText === "") {
```

Comment:
innerText가 === "" 공백일 때를 특정해서 조건을 준 것은 정말 잘했다고 생각합니다. 다만 문자열 innerText가 "" 와 같다는 것은 결국 innerText가 falsy값이라는 것을 의미하는 것 같습니다.
따라서, 다음과 같이 바꾸면 더 좋은 컨벤션을 가진 코드가 될 것으로 생각이 됩니다. 어떠신가요?
[AIRBnb 코드 컨벤션](https://github.com/tipjs/javascript-style-guide?tab=readme-ov-file#15.3)
```suggestion
    } else if (innerText) {
```

Reply:
맞습니다. 추천해주신 컨벤션이 더 간결하고 의미를 한번에 알려준다고 생각합니다. 수정하여 반영하겠습니다!